platform/tls/generic/mbedtls/src/mender-tls.c

```diff
@@ -192,6 +192,9 @@ mender_tls_init_authentication_keys(mender_err_t (*get_user_provided_keys)(char
         case MENDER_NOT_IMPLEMENTED:
             assert(false && "Unexpected return value");
             /* fallthrough */
+        case MENDER_STATE_LOOP:
+            assert(false && "Unexpected return value");
+            /* fallthrough */
```

Comment:
Why do we need to care about this case here? I don't get it.

Reply:
The smoke build threw errors that the switch had unhandled enumeration values.